src/usr/local/php/unraid-tailscale-utils/unraid-tailscale-utils/Info.php

```
@@ -471,4 +471,31 @@ public function getDNSName(): string
 
         return $this->status->Self->DNSName;
     }
+
+    public function isApprovedPeerRelay(): bool
+    {
+        $netmapRules = (array) $this->localAPI->getPacketFilterRules();
+
+        // Parse the packet filter rules to see if peer relay is approved
+        // TODO: Get a better way to do this from Tailscale
+        foreach ($netmapRules as $key => $rule) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)